undead-discord.css

```diff
@@ -36,19 +36,24 @@
 /* The following 3 blocks color the keybind-shortcut modal and buttons */
 .ddr-arrows { display: none; }
 
-.keybind-shortcut > span {
-    color: #4F545C !important;
+.keybind-shortcut > span,
+.keybind-shortcut > span > svg > g {
+    color: var(--fg-color);
     background-color: var(--secondary-bg-color) !important;
     border: none;
     box-shadow: none;
     height: 21px;
+    padding: 5px 6px;
+    fill: var(--fg-color);
 }
 
-.keybind-shortcut > span:active {
-    color: #4F545C !important;
+.keybind-shortcut > span:active,
+.keybind-shortcut > span:active > svg > g {
+    color: var(--fg-color);
     border: none;
     box-shadow: none;
     height: 21px;
+    fill: var(--fg-color) !important;
```

Comment:
You added the fill, but this just affects the svg > g component. There is no need to add fill to the `span:active" and all the other stuff to the `svg > g`. Please separate these from each other unless everything really needs to be applied to both elements.